<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDlg</class>
 <widget class="QWidget" name="ControlDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>524</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>控制设备</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Image/png/control.png</normaloff>../Image/png/control.png</iconset>
  </property>
  <property name="statusTip">
   <string>命令控制</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(77, 77, 77);color:white</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>2</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Image/png/control.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>8</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备控制</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="NameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前选择站点：</string>
     </property>
    </widget>
    <widget class="QLabel" name="StationIDLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>12</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="SrvNameLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>12</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>类型：</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>读取命令</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>88</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="R_BASEINFO_Btn">
         <property name="text">
          <string>读取基本信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="R_DATETIME_Btn">
         <property name="text">
          <string>读取时钟</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="R_LONG_Btn">
         <property name="text">
          <string>读取经度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="R_LAT_Btn">
         <property name="text">
          <string>读取纬度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="R_ALT_Btn">
         <property name="text">
          <string>读取海拔高度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="R_SNAPSHOT_Btn">
         <property name="text">
          <string>读取数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>151</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>设置命令</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>88</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="W_DATETIME_Btn">
         <property name="text">
          <string>设置时钟</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="W_LONG_Btn">
         <property name="text">
          <string>设置经度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="W_LAT_Btn">
         <property name="text">
          <string>设置纬度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="W_ALT_Btn">
         <property name="text">
          <string>设置海拔高度</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="ParamsEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>329</y>
       <width>311</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QDateTimeEdit" name="dateTimeEdit_B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>194</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_E">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>194</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="R_DMTD_Btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>86</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>补抄数据</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>5</y>
      <width>46</width>
      <height>26</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="ScrollArea_MinBtn">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>3</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_MinBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>18</height>
       </rect>
      </property>
      <widget class="QPushButton" name="MinBtn">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QScrollArea" name="ScrollArea_CloseBtn">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>3</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_CloseBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>18</height>
       </rect>
      </property>
      <widget class="QPushButton" name="CloseBtn">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
